### PR TITLE
Cart page

### PR DESCRIPTION
**Pull request by talrejah02**
please review and give suggestions

### CODE REVIEW

**Review by ShreyashMahajan — APPROVED** [review]
Good to merge

File: index.html [hunk 1]
```diff
@@ -1,12 +1,148 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (devdeadviz) [inline index.html:7]: Just a suggestion, you can also add custom meta tags for eg, `<meta name="author" content="Your Name">` to your codebase a personal touch.

**Review by SaurabhChawda — APPROVED** [review]
You can merge it.

File: index.html [hunk 2]
```diff
@@ -1,12 +1,148 @@
...
+            <section class="header-logo">
+                <section class="header-svg"><svg xmlns="http://www.w3.org/2000/svg"
+                        xmlns:xlink="http://www.w3.org/1999/xlink" aria-hidden="true" role="img"
+                        class="iconify iconify--iconoir" width="32" height="32" preserveAspectRatio="xMidYMid meet"
+                        viewBox="0 0 24 24">
+                        <path fill="none" stroke="#FFEA29" stroke-linecap="round" stroke-linejoin="round"
+                            stroke-width="1.8"
+                            d="m18 10l-3 1.5l-4-3l-1 5.5l3.5 3l.5 4.5m4-13v13M10 17l-2 4.5m.5-13C7 9.5 6 12 6 12l2 1m4-6.5a2 2 0 1 0 0-4a2 2 0 0 0 0 4Z">
+                        </path>
+                    </svg></section>
```
Comment (devdeadviz) [inline index.html:26]: If you are using something which is not generic, kindly add comments for better understanding.

File: index.html [hunk 3]
```diff
@@ -1,12 +1,148 @@
...
+                <section class="header-svg">
+                    <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"
+                        aria-hidden="true" role="img" class="iconify iconify--icon-park-outline" width="20" height="20"
+                        preserveAspectRatio="xMidYMid meet" viewBox="0 0 48 48">
+                        <g fill="none" stroke="currentColor" stroke-linejoin="round" stroke-width="4">
+                            <path d="M21 38c9.389 0 17-7.611 17-17S30.389 4 21 4S4 11.611 4 21s7.611 17 17 17Z"></path>
+                            <path stroke-linecap="round"
+                                d="M26.657 14.343A7.975 7.975 0 0 0 21 12c-2.21 0-4.21.895-5.657 2.343m17.879 18.879l8.485 8.485">
+                            </path>
+                        </g>
```
Comment (devdeadviz) [inline index.html:39]: What is the `<g>` tag?

File: pages/cart/cart.css [hunk 4]
```diff
@@ -0,0 +1,126 @@
+@import url(../../style.css);
+@import url(../product/product.css);
+
+.cart-wrapper {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    height: 77vh;
+    background-color: var(--secondary-grey);
+
```
Comment (devdeadviz) [inline pages/cart/cart.css]: remove extra spaces

File: pages/cart/cart.css [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+    justify-content: center;
+    align-items: center;
+    margin-left: auto;
+    margin-right: 10rem;
+}
+
+/* --------card--------- */
+.horizontal {
+    flex-direction: row;
+    min-width: 450px;
```
Comment (devdeadviz) [inline pages/cart/cart.css]: Try to use relative units instead of absolute units.

**Review by devdeadviz — CHANGES_REQUESTED** [review]
Everything looks good, just added some small suggestions.

**Comment by devdeadviz** [comment]
Add some information in the description about what you are adding in the given pull request along with the gif or screenshots.

File: index.html (hunk 3 above)
Comment (talrejah02) [reply index.html:39]: it came along with svg

**Pull request merged**